tests/console/system_prepare.pm

```diff
@@ -38,8 +38,8 @@ sub run {
     if (!check_var('DESKTOP', 'textmode')) {
         # Make sure packagekit is not running, or it will conflict with SUSEConnect.
         quit_packagekit;
-        # poo#77134 wait for packagekit related zypper process to exit
-        assert_script_run 'until ! pgrep zypper; do sleep 1; done';
+        # poo#77134 wait for zypper process to exit
+        wait_quit_zypper;
```

Comment:
More comments needed here to clarify why we need wait zypper

Reply:
Some zypper processes ( such as purge-kernels) in background hold the lock, usually it's not intended or common that you run 2 zypper tasks at the same time, so we need wait the zypper process in background to finish and release the lock so that we can run a new zypper for our test.

Reply:
Updated.